### DISCUSSION TITLE
ESP32 mit W5500 Ethernet Modul

### DISCUSSION THREAD

**Discussion by pr1m300** [tbnobody/OpenDTU#1006] [Q&A]
Hallo, 
ich würde gerne an meinen Standart ESP32 30 Pin Devboard ein Ethernet Anschluss nachrüsten. Ich habe in den eth.type einstellungen gesehen das bereits einige Ethernet Module unterstützt werden, wie z.b. der LAN8720. 
Ich möchte jedoch gerne aus platzgründen ein W5500 Lite Ethernet Modul nachrüsten.
Hat das jemand schonmal gemacht funktioniert das eventuell mit einen der bestehenden eth.type einstellungen?

Danke.

**Comment by Garfs-Homassistant** [comment]
Würde mich auch interessieren
